templates/admin/playsounds.html

```diff
@@ -78,7 +78,7 @@ <h2>Playsounds</h2>
                 {% if  playsounds_enabled %}
                     You are good to go! Your viewers can play the sounds listed below.
                 {% else %}
-                    Your viewers can currently not play any sounds.
+                    Your viewers currently cannot play any sounds.
```

Comment:
this one doesn't feel more correct to me, maybe a different phrasing would be better? feel free to restructure the entire sentence if it helps